content/concepts/transports/quic.md

```diff
@@ -82,6 +82,13 @@ In addition, a client can make use of QUIC's 0 RTT feature for subsequent connec
 when it has already communicated with a certain server. The client can then send
 (encrypted) application data even before the QUIC handshake has finished.
 
+### QUIC native multiplexing
+
+A single QUIC packet can include multiple frames from one or more
+streams. Since QUIC packets can be decrypted even when they're received out of order, this solves the problem of HOL (head-of-line) blocking: If a packet that contains
```

Comment:
```suggestion
streams. Since QUIC packets can be decrypted even when they're received out of order, this solves the problem of HoL (head-of-line) blocking that multiplexers applied on top of a TCP connection suffer from: If a packet that contains
```

Can we link to the HoL explanation we have (do we?) for the other muxers?

Reply:
linked to the "Key challenges with TCP" section in this document which includes an explanation.